flask_resty_tenants/authorization.py

```diff
@@ -76,6 +81,19 @@ def get_authorized_tenant_ids(self, role):
 
             yield tenant_id
 
+    def check_request_tenant_id(self):
+        try:
```

Comment:
I had that briefly – the issue is, what if somehow `tenant_id` were omitted and the value in `view_args` actually _was_ `None`? It can't happen as set up right now, but I'm not sure we wouldn't want to 404 in that case.